pandas/tests/tools/test_to_datetime.py

```diff
@@ -2443,3 +2443,22 @@ def test_na_to_datetime(nulls_fixture, klass):
     result = pd.to_datetime(klass([nulls_fixture]))
 
     assert result[0] is pd.NaT
+
+
+def test_empty_string_datetime_coerce():
+    # GH13044
+    td = Series(["03/24/2016", "03/25/2016", ""])
+    format = "%m/%d/%Y"
+
+    # coerce empty string to pd.NaT
+    result = pd.to_datetime(td, format=format, errors="coerce")
+    expected = Series(["2016-03-24", "2016-03-25", pd.NaT], dtype="datetime64[ns]")
+    pd.testing.assert_series_equal(expected, result)
+
+    # raise an exception in case a format is given
+    with pytest.raises(ValueError, match="does not match format"):
+        result = pd.to_datetime(td, format=format, errors="raise")
+
+    # don't raise an expection in case no format is given
+    result = pd.to_datetime(td, errors="raise")
```

Comment:
can you make another test for the unit cases

Reply:
added a test for the unit cases like discussed in #13044

Reply:
@jreback is it okay like that?